src/ebay-tourtip/ebay-tourtip-heading.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ComponentProps, FC } from 'react'
+import classNames from 'classnames'
+
+type Props = ComponentProps<'h2'> & {
+    type?: string
+}
+
+const EbayTooltipHeading: FC<Props> = ({
+    className,
+    type = 'infotip',
+    ...props
+}) => (
+    <h2 {...props} className={classNames(`${type}__heading`, className)} />
```

Comment:
We should not always render a `h2`. We should have an `as` prop (that defaults to `h2`) and render that instead. You can check for the [notice-title](https://github.com/eBay/ebayui-core-react/blob/815f07acab594a1a6a66c568e5abdedd7363cbd7/src/common/notice-utils/notice-title.tsx#L11) for example. This is because applications owners can have control on what type of heading can be rendered